src/main/resources/errors.yaml

```diff
@@ -393,7 +393,7 @@ request-errors:
     code: MAP_TYPE_INCORRECT_DEFINITION
     title: Map column data type definition provided in the request is incorrect.
     body: |-
-      Map column data type definition provided in the request is incorrect.
+      Map column data type definition provided in the request is incorrect: ${reason}
```

Comment:
should it be "invalid" instead of "incorrect"? Or maybe even better "unsupported".